tools/updateSelectize.R

```diff
@@ -80,4 +80,4 @@ for (patch in list.files(patch_dir, full.names = TRUE)) {
 # =============================================================================
 # Generate minified js
 # =============================================================================
-withr::with_dir(rprojroot::find_package_root_file("tools"), system("yarn build"))
+withr::with_dir(rprojroot::find_package_root_file("tools"), system("yarn grunt"))
```

Comment:
@jooyoungseo what's the reasoning for this change?

Reply:
In order to follow the instructions of `./tools/README.md`:

> After updating and applying patches, `yarn grunt` should be run per the instructions above in order to generate a minified JavaScript file.

https://github.com/rstudio/shiny/blob/master/tools/README.md#L112

> 1. Run `yarn grunt`

https://github.com/rstudio/shiny/blob/master/tools/README.md#L123